model/publishing/delivery/tasks/DeployTestEnvironments.php

```diff
@@ -160,18 +156,38 @@ private function getQtiTestPackageFile(core_kernel_classes_Resource $delivery):
     {
         $packagePath = $delivery->getOnePropertyValue($this->getProperty(TestBackupService::PROPERTY_QTI_TEST_BACKUP_PATH));
         $packagePath = (string)  $packagePath;
-        
+
         return $this->getServiceLocator()
             ->get(FileSystemService::SERVICE_ID)
             ->getDirectory(TestBackupService::FILESYSTEM_ID)
             ->getFile($packagePath);
     }
 
+    private function getDeliveryPropertyList(\core_kernel_classes_Resource $delivery): array
```

Comment:
will be better to move that to the service

```
$this->getPublishingDeliveryService()->getSyncDeliveryProperties()
```

Reply:
Implementation was moved around because of other refactorings, ended up in `RemoteDeliveryPublisher`